frontend/assets/javascripts/src/modules/analytics/ga.es6

```diff
@@ -90,6 +90,9 @@ export function wrappedGa(a,b,c){
 
 export function init() {
     let guardian = window.guardian;
+
+    console.log('Loading Google Analytics');
```

Comment:
Was this for debugging? Do you want to keep this in?

Reply:
Yes I want to keep this.

Reply:
Actually, I'll do it the same way as subs.